File: Course_01/C++ Language/EstruturaCondicional/glicose.cpp
#include <iostream>

using namespace std;

int main()
{
    double medida;

    cout << "Digite a medida da glicose: ";
    cin >> medida;

    if (medida <= 100) {
        cout << "Classificacao: normal" << endl;
    }
    else if (medida > 100 && medida <= 140) {
        cout << "Classificacao: elevado";
    }
    else {
        cout << "Classificacao: diabetes";
    }

    return 0;
}

File: Course_01/Java Language/Estrutura condicional/glicose.java
package condicional;

import java.util.Locale;
import java.util.Scanner;

public class glicose {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		double medida;
		
		System.out.print("Digite a medida da glicose: ");
		medida = sc.nextDouble();
		
		if (medida <= 100) {
			System.out.println("Classificação: normal");
		}
		else if (medida > 100 && medida <= 140) {
			System.out.println("Classificação: elevado");
		}
		else {
			System.out.println("Classificação: diabetes");
		}
		
		sc.close();
	}

}
